modular_zubbers/code/modules/clothing/armor_overrides/hecusecvarient.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/clothing/head/helmet/hecu/sec
+	armor_type = /datum/armor/head_helmet
```

Comment:
I don't really like these as-is, at the very least if HECU is obtainable. These are extra items that share the same sprite & have lower armor values but are otherwise identical to their HECU counterparts.

Reply:
Yup. I thought it'd be good for gimmicks at the least. They're, to my knowledge unobtainable outside of the gateway, bitrunning and admin spawns. They have, HoS tier stats iirc.


If you want em axed I can axe em.